tests/snappy_pipeline/workflows/test_workflows_somatic_gene_fusion_calling.py

```diff
@@ -31,7 +32,7 @@ def minimal_config():
             bwa:
               path_index: /path/to/bwa/index.fasta
           somatic_gene_fusion_calling:
-              tools: ['fusioncatcher', 'jaffa', 'arriba']
+              tools: ['arriba', 'fusioncatcher']
```

Comment:
Shouldn't the test include all available tools?
"arriba', "fusioncatcher", "jaffa", "pizzly", "hera", "star_fusion"